problemtools/verifyproblem.py

```diff
@@ -293,6 +284,15 @@ def get_subgroup(self, name):
         return next((sub for sub in self._items if isinstance(sub, TestCaseGroup) and os.path.basename(sub._datadir) == name), None)
 
 
+    def get_grader_flags_for_judging(self):
+        problem_type = self._problem.config.get('type')
+        if self.config['grading'] == 'default':
+            # The default grader flags are different depending on the problem type, which requires this override
+            default_grader_spec = ['always_accept', 'sum'] if problem_type == 'scoring' else ['no_errors', 'sum']
+            return default_grader_spec + self.config['grader_flags'].split()
```

Comment:
I wonder if this was a bad choice in the spec.  What do you think about letting the default for default_grader always be `no_errors`?  It means one has to set it in the root testdata.yaml for all IOI problems, but on the other hand one gets rid of this annoying special casing, and can also put the default value in the default_grader instead of requiring the CCS to pass a verdict aggregation type.